requirements.txt

```diff
@@ -10,8 +10,8 @@ djangorestframework==3.3.2
 edx-api-client==0.3.0
 elasticsearch==2.3.0
 elasticsearch-dsl==2.1.0
-factory_boy==2.6.0
-fake-factory==0.5.3
+factory_boy==2.7.0
```

Comment:
I needed to update it in order to use the `tzinfo` parameter of `FAKE.date_time_this_month()` function.